docs/site/todo-list-tutorial-repository.md

```diff
@@ -0,0 +1,77 @@
+---
+lang: en
+title: 'Add TodoList Repository'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-repository.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Repository
+---
+
+### Repositories with related models
+
+The repository of a model with relations can make constrained version of the
+related model's repository from the source repository's instance itself. In this
+section, we'll create `TodoListRepository` which also has the capability of
+building a constrained version of `TodoRepository`.
+
+### Create your repository
+
+In the `src/repositories` directory:
+
+- create `todo-list.repository.ts`
+- update `index.ts` to export the newly created repository
+
+Like `TodoRepository`, we'll use `DefaultCrudRepository` to extend our TodoList
+repository and inject the same datasource. From there we'll need to make two
```

Comment:
Maybe explain we're using the same datasource since we want to persist the list on the same database?